components/ledger/pkg/storage/sqlstorage/ledger/migrates/0-init-schema/postgres.sql

```diff
@@ -49,14 +47,12 @@ CREATE TABLE IF NOT EXISTS "VAR_LEDGER_NAME_v2_0_0".logs_ingestion (
 
 --statement
 CREATE TABLE IF NOT EXISTS "VAR_LEDGER_NAME_v2_0_0".logs_v2 (
-    id bigint,
+    id bigint primary key,
```

Comment:
I think we should add them in bun models too, for example:

ID             uint64    `bun:"id,unique,type:bigint"` ->
I think we should add them in bun models too, for example:

ID             uint64    `bun:"id,unique,pk,type:bigint"`